jbpm-task-assigning/jbpm-task-assigning-core/src/test/java/org/jbpm/task/assigning/BaseTaskAssigningTest.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.task.assigning;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.Set;
+
+import org.apache.commons.io.FileUtils;
+import org.jbpm.task.assigning.model.OrganizationalEntity;
+import org.jbpm.task.assigning.model.Task;
+import org.jbpm.task.assigning.model.TaskAssigningSolution;
+import org.jbpm.task.assigning.model.TaskOrUser;
+import org.jbpm.task.assigning.model.User;
+import org.optaplanner.core.api.solver.Solver;
+import org.optaplanner.core.api.solver.SolverFactory;
+import org.optaplanner.core.config.score.director.ScoreDirectorFactoryConfig;
+import org.optaplanner.core.config.solver.SolverConfig;
+import org.optaplanner.core.config.solver.termination.TerminationConfig;
+import org.optaplanner.persistence.xstream.impl.domain.solution.XStreamSolutionFileIO;
+
+public class BaseTaskAssigningTest {
+
+    public static final String _24TASKS_8USERS_SOLUTION = "/data/unsolved/24tasks-8users.xml";
+    public static final String _50TASKS_5USERS_SOLUTION = "/data/unsolved/50tasks-5users.xml";
+    public static final String _100TASKS_5USERS_SOLUTION = "/data/unsolved/100tasks-5users.xml";
+    public static final String _500TASKS_20USERS_SOLUTION = "/data/unsolved/500tasks-20users.xml";
+
+    protected boolean writeTestFiles() {
+        return Boolean.parseBoolean(System.getProperty("org.jbpm.task.assigning.test.writeFiles", "false"));
+    }
+
+    protected SolverFactory<TaskAssigningSolution> createSolverFactory() {
+        SolverFactory<TaskAssigningSolution> solverFactory = SolverFactory.createEmpty();
```

Comment:
The `createEmpty()` method has been deprecated recently, please see https://github.com/kiegroup/optaplanner/pull/577